sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -445,14 +445,13 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
     throwTypeErrorIfParameterNotLong(connId, "fromSequenceNumber", fromSequenceNumber);
 
     // Checks for maxMessageCount
-    if (maxMessageCount !== undefined) {
-      throwTypeErrorIfParameterTypeMismatch(connId, "maxMessageCount", maxMessageCount, "number");
-      if (maxMessageCount <= 0) {
-        return [];
-      }
-    } else {
+    if (maxMessageCount == undefined) {
       maxMessageCount = 1;
     }
+    maxMessageCount = Number(maxMessageCount);
```

Comment:
Why?

Reply:
Because JavaScript! For the case when someone passes the string "123"

Reply:
Shouldn't we throw in that case?

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/12453/commits/e45d0644a346d7940c78bfc6d2a5d2acec77233a